quickstart/docker/compose/docker-compose.yaml

```diff
@@ -16,22 +18,19 @@ services:
       ROUTE_GUIDE_SERVER_PORT: 50051
     volumes:
       - ../../zilla.yaml:/etc/zilla/zilla.yaml
-      - ./grpc/echo.proto:/etc/zilla/proto/echo.proto
-      - ./grpc/route_guide.proto:/etc/zilla/proto/route_guide.proto
-    command: start -v -e -Pzilla.binding.kafka.client.connection.pool=false
+      - ./proto/route_guide.proto:/proto/route_guide.proto
+    command: start -v -e -Pzilla.engine.worker.capacity=512
 
   mqtt-simulator:
-    build: https://github.com/vordimous/mqtt-simulator.git#zilla-quickstart
+    image: ghcr.io/vordimous/mqtt-simulator:zilla-quickstart
```

Comment:
Do we need to move this to extras?

Reply:
we can, I am trying to get the original dev to merge some changes so that we can just use the official image. So this is my fork with the PR into the official repo.